dbt_sugar/core/ui/cli_ui.py

```diff
@@ -169,13 +173,57 @@ def _validate_question_payload(self) -> None:
         self._is_valid_question_payload = True
 
     @staticmethod
-    def _iterate_through_columns(cols: List[str]) -> Dict[str, str]:
+    def _iterate_through_columns(
```

Comment:
I don't really understand which this is part of your code change do you know? I changed the base to be against my PR and still this shows up as if you were adding it. Can you check and get back to me?